src/Fixes/ActorIsHostileToActorFix.h

```diff
@@ -22,7 +22,11 @@ namespace Fixes::ActorIsHostileToActorFix
 	inline void Install()
 	{
 		constexpr std::size_t size = 0x10;  // minimum function size
+#ifndef FALLOUTVR
 		REL::Relocation<std::uintptr_t> target{ REL::ID(1022223) };
+#else
+		REL::Relocation<std::uintptr_t> target{ REL::Offset(0x1415df0) };
```

Comment:
This address looks wrong to me. It's too short.

Reply:
Nevermind, it's 141415df0

Reply:
yeah it's just the offset not the address